addons/survey/models/survey_question.py

```diff
@@ -541,6 +545,7 @@ def _prepare_statistics(self, user_input_lines):
                 comment_line_ids=comment_line_ids)
             question_data.update(question._get_stats_summary_data(answer_lines))
 
+
```

Comment:
(If you remove the other part, remove that blank space to avoid modifying this file).